workflow-testing/src/main/java/com/squareup/workflow1/testing/RealRenderTester.kt

```diff
@@ -259,20 +259,38 @@ internal class RealRenderTester<PropsT, StateT, OutputT, RenderingT>(
     processedAction = value
   }
 
-  override fun verifyAction(block: (WorkflowAction<PropsT, StateT, OutputT>) -> Unit) {
+  override fun verifyAction(
+    block: (WorkflowAction<PropsT, StateT, OutputT>) -> Unit
+  ): RenderTestResult<PropsT, StateT, OutputT, RenderingT> = apply {
     val action = processedAction ?: noAction()
     block(action)
   }
```

Comment:
```suggestion
  ): RenderTestResult<PropsT, StateT, OutputT, RenderingT> {
    val action = processedAction ?: noAction()
    block(action)
    return this
  }
```

Reply:
It's obfuscating. We're just implementing an interface method, how does `apply` make this more readable than simply ending a method with `return this`?

Reply:
I'm on the other side of the spectrum here — why do we need to write explicitly `return this` if Kotlin has a good convention to do it (ok, I agree that `apply` could be misleading here because we don't mutate anything). Anyway, not a hill I would die on, and you're the mastermind behind this repo :)